modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/AbstractSnapshotSelfTest.java

```diff
@@ -1079,7 +1079,7 @@ public Account(int id, int balance) {
     }
 
     /** */
-    protected static class BlockingExecutor implements Executor {
+    public static class BlockingExecutor implements Executor {
```

Comment:
It's a bad practice make the internal static classes public. Let's revert these changes, maybe it's better to inherit the test and invoke `doSnapshotCancellationTest` method?